src/python/espressomd/lb.pyx

```diff
@@ -324,7 +324,7 @@ IF LB or LB_GPU:
             self.node[0] = key[0]
             self.node[1] = key[1]
             self.node[2] = key[2]
-            if not lb_lbnode_is_index_valid(self.node):
+            if not lb_lbnode_is_index_valid(<const Vector3i&>self.node):
```

Comment:
Why is this needed ? `Vector3i` can bind to `Vector3i const &` (but not to `Vector3i const`)

Reply:
I have no idea... i'm exploring the issue

Reply:
It seems like a bug in cython but I'm not sure what I have changed to trigger it

Reply:
Did you look at the generated c++? This sometimes helps to understand what's going on.

Reply:
In part this is because your return types are `const` qualified